source/containers.tex

```diff
@@ -15032,3 +15034,2423 @@
+all other elements are equal to 0.
+\end{itemize}
+
+\pnum
+In subclauses \ref{mdspan.layout.reqmts} through \ref{mdspan.layout.stride},
+let \exposid{is-mapping-of} be the exposition-only variable template defined as follows:
+\begin{codeblock}
+template<class Layout, class Mapping>
+constexpr bool @\exposid{is-mapping-of}@ =  // \expos
+  is_same_v<typename Layout::template mapping<typename Mapping::extents_type>, Mapping>;
+\end{codeblock}
+
+\rSec3[mdspan.layout.reqmts]{Requirements}
+
+\pnum
+A type \tcode{M} meets the \defn{layout mapping} requirements if
+
+\begin{itemize}
+\item
+\tcode{M} models \libconcept{copyable} and \libconcept{equality_comparable},
+\item
+\tcode{is_nothrow_move_constructible_v<M>} is \tcode{true},
+\item
+\tcode{is_nothrow_move_assignable_v<M>} is \tcode{true},
+\item
+\tcode{is_nothrow_swappable_v<M>} is \tcode{true}, and
+\item
+the following types and expressions are well-formed and
+have the specified semantics.
+\end{itemize}
+
+\begin{itemdecl}
+typename M::extents_type
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+A type that is a specialization of \tcode{extents}.
+\end{itemdescr}
+
+\begin{itemdecl}
+\tcode{typename M::size_type}
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{typename M::extents_type::size_type}.
+\end{itemdescr}
+
+\begin{itemdecl}
+typename M::rank_type
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{typename M::extents_type::rank_type}.
+\end{itemdescr}
+
+\begin{itemdecl}
+typename M::layout_type
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+A type \tcode{MP} that meets
+the layout mapping policy requirements\iref{mdspan.layout.policy.reqmts} and
+for which \tcode{\exposid{is-mapping-of}<MP, M>} is \tcode{true}.
+\end{itemdescr}
+
+\begin{itemdecl}
+m.extents()
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{const typename M::extents_type\&}
+\end{itemdescr}
+
+\begin{itemdecl}
+m(i...)
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{typename M::size_type}
+
+\pnum
+\returns
+A nonnegative integer
+less than \tcode{numeric_limits<typename M::size_type>::max()} and
+less than or equal to \tcode{numeric_limits<size_t>::max()}.
+\end{itemdescr}
+
+\begin{itemdecl}
+m(i...) == m(static_cast<typename M::size_type>(i)...)
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{bool}
+
+\pnum
+\returns
+\tcode{true}
+\end{itemdescr}
+
+\begin{itemdecl}
+m.required_span_size()
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{typename M::size_type}
+
+\pnum
+\returns
+If the size of the multidimensional index space \tcode{m.extents()} is 0,
+then 0,
+else 1 plus the maximum value of \tcode{m(i...)} for all \tcode{i}.
+\end{itemdescr}
+
+\begin{itemdecl}
+m.is_unique()
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{bool}
+
+\pnum
+\returns
+\tcode{true} only if
+for every \tcode{i} and \tcode{j} where \tcode{(i != j || ...)} is \tcode{true},
+\tcode{m(i...) != m(j...)} is \tcode{true}.
```

Comment:
Don't know why, but the `!=` introduced a break...
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/22275534/180906626-5aeab486-d955-49d6-90a8-cbc3e8adb342.png">

Reply:
Do you mean spacing? It seems like the result of justification.

Reply:
Yeah, now I look at this again, probably this is just a spacing issue done by LaTeX engines. Nothing looks wrong on the source code itself.